### PR TITLE
Add BSVE authentication

### PR DESCRIPTION
**Pull request by nathanathan**
https://www.pivotaltracker.com/story/show/136147441

### CODE REVIEW

File: server/accounts_config.coffee [hunk 1]
```diff
@@ -1,3 +1,38 @@
...
+      headers:
+        "harbinger-auth-ticket": authInfo.authTicket
+    })
+    if Meteor.settings.private?.disableBSVEAuthentication
+      throw new Meteor.Error("BSVEAuthFailure", "BSVE Authentication is disabled.")
+    if response.data.status != 1
+      throw new Meteor.Error("BSVEAuthFailure", response.data.message)
+    meteorUser = Meteor.users.findOne(username: "bsve-" + authInfo.user)
+    if not meteorUser
+      console.log "Creating user"
```
Comment (Broham) [inline server/accounts_config.coffee]: Do we want to keep this in the master branch or was it more for debugging?

Comment (nathanathan) [inline server/accounts_config.coffee]: I want to keep in the master branch. If problems arise in production having some logging here can make it easier to track them down.

Comment (Broham) [inline server/accounts_config.coffee]: Ok - looks good then!

**Comment by Broham** [comment]
:+1:

**Pull request merged**